views/js/admin/src/services/actions/onNewLabels.ts

```diff
@@ -21,8 +21,8 @@ export function onNewLabels(response: ActionResponse<typeof modifyLabelActions[n
       });
       break;
 
-    case LabelAction.PRINT:
     case LabelAction.REFRESH:
+    case LabelAction.PRINT:
```

Comment:
Waarom is de volgorde veranderd? Hij is nu niet meer alfabetisch, en het lijkt functioneel niet uit te maken hier.